qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1157,18 +1161,35 @@ def least_busy(
         Raises:
             QiskitBackendNotFoundError: If no backend matches the criteria.
         """
-        backends = self.backends(
-            min_num_qubits=min_num_qubits, instance=instance, filters=filters, **kwargs
-        )
+        if not self._backends_list:
```

Comment:
can't comment on the actual line but I realized that the docstring is outdated (on line 1156 it uses `n_qubits`)

Reply:
I think that's a confusing example because there is a `n_qubits` property in a backend configuration so 
`least_busy(n_qubits=5, operational=True)` does work